### ISSUE TITLE
AWS EKS Support

### ISSUE THREAD

**Issue by lambertpan** [zalando-incubator/kube-ingress-aws-controller#186]
will this work with aws eks?

**Comment by hjacobs** [comment]
Yes, I think it should work (haven't tried it out yet) as it uses regular Kubernetes features (serviceAccount to connect to API server) and expects regular auto scaling group(s) or EC2 instances (which you have to operate yourself in case of EKS).

**Comment by szuecs** [comment]
I think we can close this, because the question was answered. Feel free to open an issue if it does not work on EKS.

**Comment by hjacobs** [comment]
@lambertpan I would be curious to know your experience trying it out with AWS EKS. Please let us know how it works and what the challenges are.

**Comment by lambertpan** [comment]
@hjacobs, It works very well with eks.
I followed the following steps;
1. created an IAM policy- [as in the doc](https://github.com/zalando-incubator/kube-ingress-aws-controller/blob/master/deploy/kops.md#iam-role). And attached that policy with 'eks-k8s-node' IAM role
2. Added a security group [refered the doc](https://github.com/zalando-incubator/kube-ingress-aws-controller/blob/master/deploy/kops.md#security-group-for-ingress). I used terraform, so i didnt run those commands. But followed the same steps.
3. Added the following tags to (terrafrom) worker-node auto-scaling group;
tag {
    key                 = "k8s.io/role/node"
    value               = ""
    propagate_at_launch = true
  }
This will make the controller to identify the ec2 instances.
